### PR TITLE
Remove spaces inside string template expressions

### PR DESCRIPTION
**Pull request by lionralfs**
#### Summary
This PR ensures that code samples comply with the [`template-curly-spacing`](https://eslint.org/docs/latest/rules/template-curly-spacing) ESLint rule.

#### Motivation
This is in preparation of https://github.com/orgs/mdn/discussions/143#discussioncomment-3193114 where I'm going to let ESLint automatically fix `prefer-template`. However, running `prefer-template` directly causes changes like this:

```js
console.log('Download error: ' + e.message);
```
--->
```js
console.log(`Download error: ${  e.message}`);
```

(Note the extra space inside the template expression.)
According to [this comment](https://github.com/eslint/eslint/issues/8520#issuecomment-297872379), ESLint does this on purpose:

> [...] the rule preserves spacing there so that it doesn't end up removing any comments that might appear on either side of the operator. You can use [template-curly-spacing](http://eslint.org/docs/rules/template-curly-spacing) if you want the whitespace there to be automatically removed.

Long story short, this PR fixes all *existing* violations of `template-curly-spacing` to make further PRs easier to review.

This PR…
-->
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by lionralfs** [comment]
cc @Josh-Cena @teoli2003

**Review by Josh-Cena — APPROVED** [review]
Thanks—is this everything?

**Review by teoli2003 — APPROVED** [review]

**Comment by lionralfs** [comment]
> Thanks—is this everything?

Yes, all *current* violations of `template-curly-spacing` (except files with known parsing errors, as documented [here](https://github.com/orgs/mdn/discussions/158))

**Comment by lionralfs** [comment]
I should clarify that this only refers to files in `web/api`, just as all my previous modernization effords
